includes/class-sensei-frontend.php

```diff
@@ -1237,10 +1237,22 @@ public function sensei_course_start() {
 	 * @return bool True if successful.
 	 */
 	public function manually_enrol_learner( $user_id, $course_id ) {
-		$enrolment_manager = Sensei_Course_Enrolment_Manager::instance();
-		$manual_enrolment  = $enrolment_manager->get_manual_enrolment_provider();
+		$course_enrolment = Sensei_Course_Enrolment::get_course_instance( $course_id );
 
-		return $manual_enrolment && $manual_enrolment->enrol_learner( $user_id, $course_id );
+		// If user is removed, just restore instead.
+		if ( $course_enrolment->is_learner_removed( $user_id ) ) {
+			$course_enrolment->restore_learner( $user_id );
+		}
+
+		// If user isn't still enrolled, enroll manually.
```

Comment:
I think this comment is not correct, right? Because we don't have this condition.

Reply:
Fair enough! I removed it on [43299cd](https://github.com/Automattic/sensei/pull/4848/commits/43299cd2aa424d1029bd376c57e1df0f0623bffd)! Thanks!